### ISSUE TITLE
Amazon Managed Prometheus cannot send to Chatbot to route alerts to Slack [Feature Request]

### ISSUE THREAD

**Issue by pptb-aws** [aws/amazon-managed-service-for-prometheus-roadmap#14]
At this time Amazon Managed Prometheus only sends traffic to SNS.  Chatbot only supports traffic [routed through event bridge](https://docs.aws.amazon.com/chatbot/latest/adminguide/create-eventbridge-rule.html).  Getting alerts through Slack is highly desirable as another route for notifying IT organizations that there is an issue.

**Comment by jeromeinsf** [comment]
Hi @pptb-aws 
You can go directly from SNS to Slack

**Comment by John1Q84** [comment]
Yes, Chatbot is able to subscribe SNS topic, and  you may know, Chatbot makes easier to integrate with Slack(No need to make codes with Slack SDK API)

**Comment by benabineri** [comment]
I've just tried configuring Chatbot to watch an SNS topic to which AMP Alertmanager is publishing alerts. I caused an alert to fire and got the alert through the topic via the email subscription (into a Slack channel), but all that happened with Chatbot is the UnsupportedEvents metric incremented by one.

It would be nice to have a way to get the alerts into Slack without them being emails sent via SNS to the channel's email address, as those are fiddly to share links to (among other issues). Ideally this wouldn't involve setting up a Lambda either. Does anyone have any suggestions please?
